windows_set_ovms_version.py

```diff
@@ -72,6 +72,13 @@ def get_ovms_sha():
     output = subprocess.check_output(command, shell=True, text=True)
     return output.rstrip()
 
+def check_get_product_version():
+    version = os.environ.get('PRODUCT_VERSION', OVMS_PROJECT_VERSION)
+    if version != None:
+        return version
+    else:
+        return OVMS_PROJECT_VERSION
```

Comment:
in this scenario version will always be != None (by default it's OVMS_PROJECT_VERSION), so this part of code is not needed
```suggestion
    return version
```